runtime/cni_integ_test.go

```diff
@@ -357,11 +357,16 @@ func writeCNIConf(path, chainedPluginName, networkName, nameserver string) error
 func useDefaultNetworkInterfaceRuntimeConfig(t *testing.T) {
 	t.Helper()
 
-	err := os.RemoveAll(runtimeConfigPath)
-	require.NoError(t, err, "failed to remove existing firecracker containerd runtime config file")
-
-	err = os.Symlink(defaultNetworkInterfaceRuntimeConfigPath, runtimeConfigPath)
-	require.NoError(t, err, "failed to symlink default network interface runtime config")
+	writeRuntimeConfig(func(c *Config) {
```

Comment:
I just noticed we ignore the error here, we should check it.

Reply:
Oh right. Will do.

Can't we catch that from Travis CI?

Reply:
That's a good question, I would have thought so too. A random guess is that maybe it doesn't check our test files? Don't know for sure though.

Reply:
`golangci-lint` can do that, but we have disabled the check, probably, a lot of them is about Close(), Remove() or Setenv(), ...

```
     1  firecracker-control/local.go:170: G104: Errors unhandled. (gosec)
     2  agent/drive_handler.go:93: G104: Errors unhandled. (gosec)
     3  internal/network_test_utils.go:149: G104: Errors unhandled. (gosec)
     4  internal/common_test.go:105: G104: Errors unhandled. (gosec)
     5  internal/vm/task.go:177: G104: Errors unhandled. (gosec)
     6  internal/vm/task.go:219: G104: Errors unhandled. (gosec)
     7  internal/vm/vsock.go:257: G104: Errors unhandled. (gosec)
     8  internal/vm/vsock.go:280: G104: Errors unhandled. (gosec)
     9  internal/vm/task_test.go:180: G104: Errors unhandled. (gosec)
    10  internal/vm/task_test.go:223: G104: Errors unhandled. (gosec)
    11  internal/vm/task_test.go:227: G104: Errors unhandled. (gosec)
    12  internal/vm/task_test.go:231: G104: Errors unhandled. (gosec)
    13  internal/vm/task_test.go:263: G104: Errors unhandled. (gosec)
    14  internal/vm/task_test.go:272: G104: Errors unhandled. (gosec)
    15  internal/vm/task_test.go:281: G104: Errors unhandled. (gosec)
    16  runtime/service.go:457: G104: Errors unhandled. (gosec)
    17  runtime/config_test.go:73: G104: Errors unhandled. (gosec)
    18  runtime/config_test.go:76: G104: Errors unhandled. (gosec)
    19  runtime/drive_handler_test.go:39: G104: Errors unhandled. (gosec)
    20  snapshotter/devmapper/pool_device_test.go:107: G104: Errors unhandled. (gosec)
    21  snapshotter/devmapper/pool_device_test.go:126: G104: Errors unhandled. (gosec)
```

The newer version of golangci-lint with errcheck (instead of gosec) seems less noisy. Will take a look after this change.

Reply:
https://github.com/firecracker-microvm/firecracker-containerd/pull/293 will catch the issue in future.